content/docs/source-git/work-with-source-git/build-locally.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Build current checkout locally"
+date: 2021-01-27T23:59:59+01:00
+draft: false
+weight: 3
+---
+
+# Build your current checkout locally
+
+Once you are comfortable with your local changes, you can build them before
+pushing them out. Just make sure that all code changes are committed, otherwise
+packit won't be able to create patch files out of them.
+
+We are going to use packit to achieve such a thing, so please make sure you have it
+[installed locally](/docs/guide/#have-packit-tooling-installed-locally).
+
+The command to create source RPMs is called `srpm` and that's how you can create one:
+```
+$ packit srpm
+SRPM: /home/tt/g/systemd/systemd-stable/systemd-247.1-2.g68d22b32.fc33.src.rpm
+```
+
+Our changes worked! We can try building them, either in our local environment
+using packit - this implies that all build dependencies are installed:
+```
+$ packit local-build
+```
+
+...or in [mock](https://github.com/rpm-software-management/mock):
```

Comment:
:thinking:

Would it make sense to wrap this into `packit local-build`?

Reply:
yup, sounds good!

Reply:
https://github.com/packit/packit/issues/1096